lib/Doctrine/DBAL/Driver/PDOConnection.php

```diff
@@ -7,12 +7,16 @@
 
 /**
  * PDO implementation of the Connection interface.
- * Used by all PDO-based drivers.
  *
- * @since 2.0
+ * Used by all PDO-based drivers.
  */
-class PDOConnection extends PDO implements Connection, ServerInfoAwareConnection
+class PDOConnection implements Connection, ServerInfoAwareConnection
 {
+    /**
+     * @var PDO
+     */
+    private $conn;
```

Comment:
Suggest renaming this to `connection` or even `pdoConnection` to avoid abbreviation and confusion overall